src/ratekeeper.py

```diff
@@ -357,18 +371,25 @@ def update_DRVBusy(self, mcc):
                 #busy counts cant have it yet.
 
             if drive['volume']:
+                Trace.trace(10, 'CALLING inquire_vol %s' % (drive['volume'],))
```

Comment:
I thought you wanted to get rid of calling inquire_vol?

Reply:
I did, but decided to leave as I see it working, although now always good.